### DISCUSSION TITLE
The Update today....

### DISCUSSION THREAD

**Discussion by derspanier1** [vladmandic/sdnext#1686] [General]
brought such an incredible quality increase. i am absolutely shocked how good everything suddenly looks. 
Thank you very much, Count Vlad. You are doing incredible work.

**Comment by Tillerz** [comment]
Uh yes, dear lordylord, what changed? :D I see a second pass now. :-o

dreamshaper 7 model with prompt + negative prompt from this image: https://civitai.com/images/1548627?modelVersionId=117290&prioritizedUserIds=1650619&period=AllTime&sort=Most+Reactions&limit=20

Result:
![pic](https://images.plurk.com/1w7PGvZocmHemKHIfkfUPn.jpg)
